src/back-link/_back-link.scss

```diff
@@ -22,10 +22,18 @@
     // Underline is provided by a bottom border
     text-decoration: none;
 
+    // Override link colour to use text colour
+    //
+    // GOV.UK Template includes a specific a:link:focus selector designed to
```

Comment:
Instead of 'GOV.UK Template' should we use 'alphagov/govuk_template' which is less ambiguous for people in the future?

If we intend to deprecate GOV.UK Template there will be people in the future that don't even know what this is.